code/client/iconimporter

```diff
@@ -209,13 +214,25 @@ def main():
         repo_path = arguments[0].rstrip("/")
 
     # Make sure the repo path exists
-    if not os.path.exists(repo_path):
+    if len(arguments) > 1:
+        repo_url = arguments[1].rstrip("/")
+    else:
+        repo_url = pref('repo_url')
+        if not repo_url:
+            print_err_utf8("Need to specify a URL for the repo!")
+            exit(-1)
+        else:
+            print_utf8("Using repo url: %s" % repo_url)
+
+    # Make sure the repo exists
+    repo = Repo.Open(repo_path, repo_url)
```

Comment:
Repo.Open() is defined with three parameters: path, url, and plugin. If you intend the plugin to be optional, it should be defined differently.